### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by chuilam** [chuilam/Tafe230241#1] [General]
3 questions and 3 responses

**Comment by chuilam** [comment]
Hi, Can you please confirm which feature you are working on?

**Reply by StephenHillenbrand** [reply]
I am creating the Currency Convertor, should be completed soon

**Comment by chuilam** [comment]
I am confused! Do we create the feature branches first then work on the coding? and then add that to the branches?

**Reply by StephenHillenbrand** [reply]
yes that is correct, my understanding is we create these based on the development branch that has the main menu included.

**Comment by chuilam** [comment]
Can you please let me know when you are merging your feature branch to the development branch and when its done?

**Reply by StephenHillenbrand** [reply]
Happy to let you know. will email you.
